test/integration2/query/select/where.spec.js

```diff
@@ -632,7 +639,13 @@ describe('Where', function () {
 
         it("doesn't find data using whereLike when different case sensitivity", async () => {
           const result = await knex('accounts').whereLike('email', 'Test1%');
-          expect(result).to.deep.equal([]);
+          // sqlite only supports case-insensitive search
+          if (isSQLite(knex)) {
```

Comment:
I would split this into 2 tests, as the test name no longer represents the test behavior when in SQLite

Reply:
It's done @rluvaton